tests/ert_tests/all/plugins/test_export_misfit.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+
+from ert_shared.hook_implementations.workflows.export_misfit import ExportMisfitDataJob
+from ert_shared.plugins import ErtPluginManager
+from res.enkf import EnKFMain
+
+
+def test_export_misfit(setup_case, snapshot):
+    res_config = setup_case("local/snake_oil", "snake_oil.ert")
+    ert = EnKFMain(res_config)
+    ExportMisfitDataJob(ert).run()
+    result = pd.read_hdf("misfit.hdf")
+    snapshot.assert_match(
+        result.to_csv(),
+        "csv_data.csv",
+    )
+
+
+def test_loaded():
```

Comment:
Because of the current setup, the test names all get mashed into one namespace so it needs to be longer:
```python
def test_export_misfit_data_job_is_loaded():
```